pkg/kubectl/cmd/set/set_resources.go

```diff
@@ -294,3 +294,14 @@ func (o *SetResourcesOptions) Run() error {
 	}
 	return utilerrors.NewAggregate(allErrs)
 }
+
+// SuggestedPodTemplateResources returns a list of resource types that declare a pod template
+func (o *SetResourcesOptions) SuggestedPodTemplateResources() []schema.GroupResource {
```

Comment:
this doesn't appear to be used by anything outside of this package - make private